students/17th/raon/hyeyoon/user/views.py

```diff
@@ -0,0 +1,45 @@
+import json
+from django.views           import View
+from django.http            import JsonResponse, HttpResponse
+from django.core.validators import validate_email
+from django.core.exceptions import ValidationError
+
+from .models import Account
+
+PASSWORD_MIN_LENGTH = 8
+class UserSignUpView(View):
```

Comment:
```suggestion
import json

from django.views           import View
from django.http            import JsonResponse, HttpResponse
from django.core.validators import validate_email
from django.core.exceptions import ValidationError

from .models import Account

PASSWORD_MIN_LENGTH = 8

class UserSignUpView(View):
```
요렇게 해주시는게 좋을 것 같습니다~!